faq.md

```diff
@@ -10,6 +34,44 @@ ocrd workspace --help
 ocrd workspace add --help
 ```
 
-## How do I turn off logging
+### How can I specify parameters on the command line?
+
+### How to configure logging?
+
+## OCR-D module project software
+
+### Where can I find official OCR-D module project software?
+
+### Which third-party OCR-D-compatible software exists?
+
+### Which processors are available?
+
+## Workflows and processors
+
+### How can I define workflows?
+
+### Where can I find sample workflows to experiment with?
+
+### How to handle failing workflows?
+
```

Comment:
Why does OCR-D need METS files? How can I process images without METS?

Reply:
Not possible, not desired. The METS file is actually needed but rather provided by the framework.

Reply:
Feel free to add directly to the PR, I will clean up before merge.